### DISCUSSION TITLE
Does the storage locker not work with this Mod?

### DISCUSSION THREAD

**Discussion by JosephF1** [Hadi77KSA/Plutonium-T6-Any-Player-EE-Scripts-For-Speedrunning#14] [Q&A] [answered]
?

**Comment by Hadi77KSA** [comment, marked as answer]
The scripts don’t interfere with the locker/fridge, thus it should work, but you might’ve experienced an issue with Plutonium not saving stats. If you’d like to follow an extra measure in hopes of getting the stats to save, execute the following console command before unloading the mod and/or closing the game:
```
updategamerprofile;uploadstats
```

**Reply by JosephF1** [reply]
Cheers
